minecraft_servers/crystopia/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+    "server_name": "crystopia",
+    "nice_name": "Crystopia",
+    "direct_ip": "crystopia.net",
+    "supported_languages": [
+       "en",
+       "de"
+    ],
+    "social": {
+      "web": "https://www.crystopia.net",
+      "web_shop": "https://shop.crystopia.net",
+      "web_support": "https://www.crystopia.net/wiki",
```

Comment:
It's literally the same content as the landing/home page. I think you mean https://www.crystopia.net/index.php?route=/wiki/ instead.

Reply:
Yes my bad sorry so much